### DISCUSSION TITLE
OSError: libnvrtc.so.9.0 not found

### DISCUSSION THREAD

**Discussion by TianMingChen** [apache/incubator-mxnet#17072] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20785298/70874475-41cebf00-1fed-11ea-9cce-d68b65f21cc5.png)
OSError

**Comment by wkcn** [comment]
Hi @TianMingChen , could you please provide a minimal reproduce example?

**Comment by TianMingChen** [comment]
> Hi @TianMingChen , could you please provide a minimal reproduce example?

just laike this
![image](https://user-images.githubusercontent.com/20785298/70874522-704c9a00-1fed-11ea-80a7-2d4535a38b74.png)

**Comment by wkcn** [comment, marked as answer]
`libnvrtc.so.9.0` is a library of CUDA 9.0 Toolkit. Did you install CUDA 9.0 ?

**Comment by TianMingChen** [comment]
> `libnvrtc.so.9.0` is a library of CUDA 9.0 Toolkit. Did you install CUDA 9.0 ?

yes,like this
![image](https://user-images.githubusercontent.com/20785298/70875347-a17a9980-1ff0-11ea-9c6d-013271a87313.png)

**Comment by wkcn** [comment]
Could you please try to run `python3 demo.py` rather than `sudo python3 demo.py` ?
I think the environment variables are different between root user and non-root user.

**Comment by TianMingChen** [comment]
> Could you please try to run `python3 demo.py` rather than `sudo python3 demo.py` ?
> I think the environment variables are different between root user and non-root user.

if I don't use sudo in this computer,it will have this problem in every code,I will try it in another computer.
![image](https://user-images.githubusercontent.com/20785298/70876677-75ade280-1ff5-11ea-9b53-b45a53777998.png)

**Comment by wkcn** [comment]
> > Could you please try to run `python3 demo.py` rather than `sudo python3 demo.py` ?
> > I think the environment variables are different between root user and non-root user.
> 
> if I don't use sudo in this computer,it will have this problem in every code
> ![image](https://user-images.githubusercontent.com/20785298/70876677-75ade280-1ff5-11ea-9b53-b45a53777998.png)

You can re-compile the NMS operator in SNIPER with non-root user.

**Comment by TianMingChen** [comment]
> > > Could you please try to run `python3 demo.py` rather than `sudo python3 demo.py` ?
> > > I think the environment variables are different between root user and non-root user.
> > 
> > 
> > if I don't use sudo in this computer,it will have this problem in every code
> > ![image](https://user-images.githubusercontent.com/20785298/70876677-75ade280-1ff5-11ea-9b53-b45a53777998.png)
> 
> You can re-compile the NMS operator in SNIPER with non-root user.

Ok,I try it.
